test/e2e/explainer/test_art_explainer.py

```diff
@@ -34,6 +35,7 @@
 kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
 
 
+@pytest.mark.skip
```

Comment:
Is there any issue with the art explainer test?

Reply:
Sorry disabled it since explainer images were fairly big and tests were running out of disk space.

Will re-enable it.